platformio.ini

```diff
@@ -195,325 +173,295 @@ build_flags =
   -DMIMETYPE_MINIMAL
   ; other special-purpose framework flags (see https://docs.platformio.org/en/latest/platforms/espressif8266.html)
   ; decrease code cache size and increase IRAM to fit all pixel functions
-  -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
-  ; -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
+  -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
+  ; -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
 
 lib_deps =
-  #https://github.com/lorol/LITTLEFS.git
+  ${env.lib_deps}
   ESPAsyncTCP @ 1.2.2
   ESPAsyncUDP
-  ${env.lib_deps}
 
 [esp32]
-#platform = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.2.3/platform-espressif32-2.0.2.3.zip
-platform = espressif32@3.5.0
-platform_packages = framework-arduinoespressif32 @ https://github.com/Aircoookie/arduino-esp32.git#1.0.6.4
-build_flags = -g
```

Comment:
What is the problem with '-g'? firmware.bin seems to be 100% the same with or without, however firmware.elf (needed for stackdumps) would be missing parts that make crash dumps readable.

Reply:
yeah I noticed too that there is another -ggdb flag coming from somewhere. My intention here was to make the release build smaller by removing debug symbols  and putting them in the debug_flags so that you can manually add them when debugging and not when building a release for publishing.
To achieve that, we would also have to unflag -ggdb.
However, I don't actually know if the debug information is actually in the final binary or if it is stripped by a later build step.

Reply:
Debug information is stripped from the binaries uploaded to the devices in the step that creates .bin from .elf.  The binaries would be way way way too big otherwise.